<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <title>Schizophrenia Paper</title>
        <link rel="stylesheet" href="patients.css">
    </head>
    <body>
        <div class="topnav"> 
          <a href="https://zaudrey.github.io/schizoproject.github.io/">Home</a>
          <a class"active" href="https://zaudrey.github.io/schizoproject.github.io/everyone.html">Patients</a>
        </div>
        
          <div id="mySidenav" class="sidenav">
            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
            <a class"active" href="https://zaudrey.github.io/schizoproject.github.io/everyone.html">Everyone</a>
            <a href="https://zaudrey.github.io/blog_2_ATAC+RNA.html">Treated</a>
            <a href="https://zaudrey.github.io/blog_3_scATAC.html">Untreated</a>
            <a href="https://zaudrey.github.io/blog_3_scATAC.html">Age</a>
          </div>

         <span style="font-size:30px;cursor:pointer" onclick="openNav()">&#9776; open</span>

        <div>
            <h1>Patients (Everyone)</h1>
            <h2>What is schizophrenia?</h2>
            <p>Schizophrenia has traditionally been viewed as a genetic disorder, with heritability rates estimated at ~73%7. However, previous genome-wide association studies 
                (GWAS) clearly showed a relatively low number of genetic regions associated with schizophrenia risk – these include 108 loci in the first study1 that have been 
                expanded to over 180 regions2–4. Most of these genetic variants were located in non-coding regions and hence, with a few exceptions there is little evidence 
                supporting that coding variants contribute to schizophrenia risk, which also suggests that genetic factors do not seem to be sufficient to fully explain the 
                molecular causes underlying this severe psychiatric condition.</p>

           <p>Twin studies have provided evidence that environmental factors contribute to schizophrenia susceptibility. Thus, it had originally been suggested that monozygotic
               twins, whose DNA sequences are ~100% identical, have a concordance for schizophrenia of nearly 50%8,9, yet the most recent work reported a probandwise concordance
               rate of 33% in monozygotic twins and 7% in dizygotic twins10. While such results indicate an important contribution of genetic factors to the etiology of 
               schizophrenia, they argue for a significant role of environmental events in the development of this complex disease. This concept is further supported by 
               epidemiological studies suggesting that prenatal environmental insults, such as maternal infection11 and severe adverse life events12, increase the risk of 
               schizophrenia in the offspring.</p>

           <p>Gene expression is regulated by the ability of the transcriptional machinery to access DNA, which is tightly packed into chromatin. The status of chromatin 
               organization depends on epigenetic factors, such as DNA methylation and histone modifications that primarily occur on amino-terminal tails13,14. Hence these 
               epigenetic mechanisms lead to stable changes in gene expression that are mediated via altered chromatin structure without modification of DNA sequence, and remain
               largely plastic throughout all periods of brain development and aging. It is then tempting to speculate that epigenetic mechanisms mediate, at least in part, the 
               effects of environmental factors on CNS gene activity, and are therefore potentially involved in the pathophysiology of schizophrenia and other mental 
               illnesses.</p>

           <p>Supporting this concept, previous studies reported alterations in chromatin structure and accessibility in tissue samples from schizophrenic subjects and controls.
               Most of these previous reports, however, were focused on DNA methylation differences in peripheral blood15 and brain16. Using the assay for transposase-accessible
               chromatin sequencing (ATAC-seq) in bulk tissue homogenates of postmortem frontal cortex samples, only a few differences in chromatin accessibility were observed 
               between schizophrenia subjects and controls, in contrast to thousands of age-related differential accessible chromatin regions17. Although interesting, the 
               ATAC-seq approach does not provide the same chromatin state information as histone modifications mapped by chromatin immunoprecipitation sequencing (ChIP-seq) 
               assays do. As an example, it has been suggested that certain histone marks important in cell differentiation are not reflected by ATAC-seq chromatin accessibility
               data analysis18.</p>

          <p>Recent work combined fluorescence-activated cell sorting (FACS) of neuronal and non-neuronal cell nuclei with histone ChIP-seq assays in two brain regions 
              (prefrontal cortex and anterior cingular cortex) from postmortem brain samples of subjects without any known neurological or psychiatric disease19. Besides the 
              identification of cell and region-specific histone modification landscapes in this cohort of control subjects, these findings also compared their datasets with 
              previous GWAS of individuals with psychiatric conditions, reporting that strong specific enrichments occurred with schizophrenia and weaker associations with 
              depression in both histone H3 acetylation of lysine 27 (H3K27ac) and histone H3 trimethylation of lysine 4 (H3K4me3) peaks. This correlation was almost exclusively
              observed in neuronal chromatin, but not in non-neuronal cell nuclei. Unfortunately, no study so far has performed epigenomic and transcriptomic profiling in with 
              cell type-specificity of individuals with schizophrenia and controls. Additionally, these previous epigenomic studies in postmortem human brain samples did not 
              address the potential effect of previous exposure to antipsychotics on regulation of chromatin state. This is particularly relevant considering that repeated 
              administration of antipsychotic medications leads to epigenetic modifications at selected gene regions in mouse 20,21 and postmortem human brain22,23 samples.</p>

           <p>Combining MOWChIP-seq24,25 and Smart-seq226 for low-input profiling of histone modifications and transcriptomes, respectively, here we present the first dataset 
                with cell type-specific epigenomic and transcriptomic landscapes in postmortem frontal cortex samples from two cohorts of schizophrenics either previously 
                treated or not with antipsychotic medications and control subjects individually matched by sex and age.</p>
           
            <p>Q: do i have to cite^?</p>
        </div> 
        
    </body>
</html>
